### PR TITLE
Provides token to retry connection

### PR DESCRIPTION
**Pull request by arsafiso**
It provides the token so that when a recurring event is being synced and the token expires, the system can refresh the token and continue syncing.

### CODE REVIEW

File: src/Http/Request.php [hunk 1]
```diff
@@ -92,7 +92,7 @@ public function getEvent(string $url, array $params = []): ResponseInterface
 
         $args = [
             'skipQueryParams' => $params['skipQueryParams'] ?? true,
-            'token' => !empty($params['token']) ? $params['token'] : '',
+            'token' => !empty($params['token']) ? $params['token'] : [],
```
Comment (frets1700) [inline src/Http/Request.php:95]: isn't params['token'] a string entity?

**Review by frets1700 — CHANGES_REQUESTED** [review]
Please make sure test passes, ty

File: src/Http/Request.php (hunk 1 above)
Comment (arsafiso) [reply src/Http/Request.php:95]: No, is an array with clientID, clientSecret, refreshToken etc, that we use to refresh the token on src/Http/Request->getHeadersWithToken();

**Comment by arsafiso** [comment]
Closing this PR to create a new one with the Fork updated.

**Pull request closed without merging**